### DISCUSSION TITLE
Gaze bearing and strength

### DISCUSSION THREAD

**Discussion by saeidn95** [vladmandic/human#435] [Q&A]
Looking the line line of the code here https://github.com/vladmandic/human/blob/62396317f5c31b2ec5bc7de2e9eb53b8978b9f28/src/draw/face.ts#L90
it is clear to see the role of bearing and strength as polar coordinate with y axis as the reference axis.
Approximating eye to be a sphere, and arc line on the eye surface as a straight line; if I where to express the gaze direction as two separate angles. I would break into two components (in the unit of radian) as:
  [(Math.sin(f.rotation.gaze.bearing) * f.rotation.gaze.strength),
  (Math.cos(f.rotation.gaze.bearing) * f.rotation.gaze.strength]  
I might have to divide this vector by a factor of 2, to account for the D=2R for the sphere.

**Comment by vladmandic** [comment]
yup, that's another way of doing it, why not...
